osu.Game/Rulesets/Objects/HitObject.cs

```diff
@@ -210,7 +210,7 @@ public IList<HitSampleInfo> CreateSlidingSamples()
         /// </summary>
         /// <param name="sampleName">The name of the sample.</param>
         /// <returns>A populated <see cref="HitSampleInfo"/>.</returns>
-        protected HitSampleInfo GetSampleInfo(string sampleName = HitSampleInfo.HIT_NORMAL)
+        public HitSampleInfo GetSampleInfo(string sampleName = HitSampleInfo.HIT_NORMAL)
```

Comment:
So you mention in the OP that this is partially covered by https://github.com/ppy/osu/commit/88d840a60d143cd404c6f033380653abc7550055, but that commit is doing something about volume too? Is volume also broken? @OliBomby mind elaborating?

Reply:
I added 100 volume too, because this method is supposed to easily make a playable sample based on the hitobject and the default volume is 0. I think it's good to add that here in case you use this on a hit object with no samples and you dont want samples to have 0 volume.

Reply:
Is it not weird that the default volume in the `HitSampleInfo` constructor is `0`? Maybe we should change that instead...

Reply:
I've changed the `ctor` instead.